### PR TITLE
openjdk17: 17.0.1 -> 17.0.3 (darwin)

### PR DESCRIPTION
**Pull request by rtimush**
###### Description of changes

Similar to https://github.com/NixOS/nixpkgs/pull/169581, but for darwin.

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [x] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by zarthross — APPROVED** [review]

File: pkgs/development/compilers/openjdk/darwin/17.nix [hunk 1]
```diff
@@ -29,7 +29,8 @@ let
     version = dist.jdkVersion;
 
     src = fetchurl {
-      url = "https://cdn.azul.com/zulu/bin/zulu${dist.zuluVersion}-ca-jdk${dist.jdkVersion}-macosx_${dist.arch}.tar.gz";
+      url =
+        "https://cdn.azul.com/zulu/bin/zulu${dist.zuluVersion}-ca-jdk${dist.jdkVersion}-macosx_${dist.arch}.tar.gz";
```
Comment (SuperSandro2000) [inline pkgs/development/compilers/openjdk/darwin/17.nix:33]:
```suggestion
      url = "https://cdn.azul.com/zulu/bin/zulu${dist.zuluVersion}-ca-jdk${dist.jdkVersion}-macosx_${dist.arch}.tar.gz";
```

File: pkgs/development/compilers/openjdk/darwin/17.nix [hunk 2]
```diff
@@ -68,14 +69,13 @@ let
     # fixupPhase is moving the man to share/man which breaks it because it's a
     # relative symlink.
     postFixup = ''
-      ln -nsf ../zulu-${lib.versions.major version}.jdk/Contents/Home/man $out/share/man
+      ln -nsf ../zulu-${
+        lib.versions.major version
+      }.jdk/Contents/Home/man $out/share/man
```
Comment (SuperSandro2000) [inline pkgs/development/compilers/openjdk/darwin/17.nix:74]:
```suggestion
      ln -nsf ../zulu-${lib.versions.major version}.jdk/Contents/Home/man $out/share/man
```

File: pkgs/development/compilers/openjdk/darwin/17.nix [hunk 3]
```diff
@@ -68,14 +69,13 @@ let
...
-    passthru = {
-      home = jdk;
-    };
+    passthru = { home = jdk; };
 
     meta = import ./meta.nix lib;
   };
-in
-jdk
+in jdk
```
Comment (SuperSandro2000) [inline pkgs/development/compilers/openjdk/darwin/17.nix:81]: Also please revert this.

**Comment by rtimush** [comment]
@SuperSandro2000 I reverted unintended all formatting changes

**Pull request merged**